src/settings/profile.hpp

```diff
@@ -101,7 +101,41 @@ class Profile
 
         if (game == btu::Game::FNV)
         {
-            profile.bsa_make_overrides = true;
+            profile.bsa_allow_compression = false;
+            profile.bsa_make_overrides    = true;
+
+            profile.base_per_file_settings_.pack = false;
+
+            auto fnv_lod_textures         = PerFileSettings::make_base(game);
+            fnv_lod_textures.pattern      = Pattern(u8".*lod(_[nsdgpe])?\\.dds", Pattern::Type::Regex);
+            fnv_lod_textures.tex_optimize = OptimizeType::None;
+
+            auto fnv_ui_textures         = PerFileSettings::make_base(game);
+            fnv_ui_textures.pattern      = Pattern(u8"textures/interface/.*\\.dds", Pattern::Type::Regex);
+            fnv_ui_textures.tex_optimize = OptimizeType::None;
+
+            auto fnv_dds         = PerFileSettings::make_base(game);
+            fnv_dds.pattern      = Pattern(u8".*\\.dds", Pattern::Type::Regex);
+            fnv_dds.tex.compress = true;
```

Comment:
Why? FNV only supports older compression formats that create visual artifacts. Compression should probably not be the default

Reply:
When approaching the default profile I tried to set it up with what I perceive to be the “golden standard” for FNV mods in the community. Which as of now is to use compressed textures at most 2k in resolution. 

Many retexture mods are hilariously oversized (4k+), sometimes even uncompressed. Interestingly nearly all textures >2k are mipmapped by the engine to <2k so using them has no benefit at all.